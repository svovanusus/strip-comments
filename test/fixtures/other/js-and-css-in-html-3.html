<script>
  console.log(1); // Comment

  var x = '<style>'
  // <style>
  (`<style>`);
  /* This is style 2 */
  document.styleSheets += `.class {
    background: url(https://some.site/url/file.jpg);
  }`;
  var obj = {
    https://some.site/path/file.jpg,
  };
  const y = "</style>";

</script>
