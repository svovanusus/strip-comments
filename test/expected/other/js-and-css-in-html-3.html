<script>
  console.log(1); 

  var x = '<style>'
  
  (`<style>`);
    document.styleSheets += `.class {
    background: url(https://some.site/url/file.jpg);
  }`;
  var obj = {
    https:
  };
  const y = "</style>";

</script>
